src/hooks.d.ts

```diff
@@ -67,8 +67,8 @@ export interface ModelHooks<M extends Model = Model, TAttributes = any> {
 
 export interface SequelizeHooks<
   M extends Model<TAttributes, TCreationAttributes> = Model,
-  TAttributes = any,
-  TCreationAttributes = TAttributes
+  TAttributes extends {} = any,
+  TCreationAttributes extends {}= TAttributes
```

Comment:
Tiny spacing issue here

```suggestion
  TCreationAttributes extends {} = TAttributes
```

Reply:
Fixed!